yaksh/test_models.py

```diff
@@ -952,7 +953,11 @@ def setUpClass(self):
         self.profile = self.user.profile
         self.quiz = Quiz.objects.get(description='demo quiz 1')
         self.question_paper = QuestionPaper(quiz=self.quiz, total_marks=3)
+        self.question_paper2 = QuestionPaper(quiz=self.quiz, total_marks=3)
+        self.question_paper3 = QuestionPaper(quiz=self.quiz, total_marks=3)
```

Comment:
Also check whether the marks are proper.
I mean marks=3 means that the question paper should create an answer paper with 3 marks total.